static/src/javascripts/projects/common/views/experiments/recommended-for-you-opt-in.html

```diff
@@ -0,0 +1,44 @@
+<section id="recommended-for-you" class="fc-container fc-container--has-show-more js-container--toggle" data-component="recommended-for-you" data-link-name="recommended-for-you" aria-expanded="true">
+    <div class="fc-container__inner">
+        <div class="fc-container__header js-container__header ">
+            <div class="fc-container__header__title">
+                <%= profileIcon %>
+                <span class="fc-container__title__text">recommended <span class="rfy-light-blue">for&nbsp;you</span></span>
+            </div>
```

Comment:
The container is missing some spacing at the top on widescreen view. Think it is due to the lack of the show/hide toggle button:

<img width="1198" alt="picture 441" src="https://cloud.githubusercontent.com/assets/8607683/21681461/37052258-d347-11e6-90da-23547cad25b7.png">


Quick fix (if desired) is to insert this:
```
<button class="fc-container__toggle" data-link-name="Hide">
    <span class="inline-arrow inline-icon"><svg width="24" height="10" viewBox="0 0 24 10"><path d="M0 .7l10.5 8.4 1.1.9 1.1-.9L23.2.7l-.5-.7-11.1 6.3L.6 0 0 .7z"></path></svg></span>
    <span class="fc-container__toggle__text">Show</span>
</button>
```

<img width="1187" alt="picture 442" src="https://cloud.githubusercontent.com/assets/8607683/21681451/2e69f524-d347-11e6-8453-82ac08dc03d7.png">

<img width="1169" alt="picture 443" src="https://cloud.githubusercontent.com/assets/8607683/21681545/8e2fb7dc-d347-11e6-911c-956b5e6eed16.png">